search-service/src/test/resources/ngsild/permission/permission.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "urn:ngsi-ld:Permission:1",
+  "type": "Permission",
+  "target":{
+          "id": "urn:ngsi-ld:Vehicle:A456",
+          "type": "Vehicle"
+        },
+  "assignee": "thomas.bousselin@egm.io",
```

Comment:
don't set your real email address in the code. also, assignee is expected to be a sub so this can be confusing.